backend/src/main/kotlin/app/ehrenamtskarte/backend/mail/Mailer.kt

```diff
@@ -55,12 +56,12 @@ object Mailer {
                         .from(fromName, smtpConfig.username)
                         .withSubject(subject)
                         .withPlainText(message)
-                        .withHeader("Content-Type", "text/plain; charset=UTF-8")
+                        .withHTMLText(messageHTML)
```

Comment:
I think this header should not be removed, is it moved somewhere or have you deleted it?

Reply:
the header will be calculated automatically depending on what you send and which charset your backend uses.

Reply:
I believe a suitable header is already added automatically by the email library

Reply:
But it doesn't hurt to double check :)

Reply:
yes i checked it.